static/app/views/issueDetails/groupFeatureFlags/useGroupFeatureFlags.tsx

```diff
@@ -0,0 +1,45 @@
+import {defined} from 'sentry/utils';
+import {
+  type ApiQueryKey,
+  useApiQuery,
+  type UseApiQueryOptions,
+  type UseApiQueryResult,
+} from 'sentry/utils/queryClient';
+import type RequestError from 'sentry/utils/requestError/requestError';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {GroupTag} from 'sentry/views/issueDetails/groupTags/useGroupTags';
+
+interface FetchIssueFlagsParameters {
```

Comment:
```suggestion
interface FetchGroupFlagsParameters {
```

nit; to match the hook name